Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift

```diff
@@ -42,11 +42,7 @@ class MerchantResultViewController: UIViewController {
         if logs.count > 0 {
             if let data = try? JSONSerialization.data(withJSONObject: logs) {
                 if let prettyNSStr = data.prettyPrintedJSONString {
-                    if let prettyStr = prettyNSStr as? String {
```

Comment:
This is needed in the case of some failures of WebDriverIO (used in UI tests).

Reply:
reversed.